File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int row = accounts.size();
        int col = accounts[0].size();
        int maxSum = INT_MIN ;
        
        for(int i=0; i < row ; i++) {
            int sum = 0;
            for(int j =0; j<col ; j++) {
                sum += accounts[i][j];
            }
            if(sum > maxSum) maxSum = sum;
        }
        return maxSum;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution(object):
    def maximumWealth(self, accounts):
        """
        :type accounts: List[List[int]]
        :rtype: int
        """
        maxsum = float('-inf')
        
        for row in accounts:
            current_sum = sum(row)
            if current_sum > maxsum:
                maxsum = current_sum  
                
        return maxsum
        